.github/workflows/package-debian.yml

```diff
@@ -64,19 +66,45 @@ jobs:
       - name: Build
         id: build
         env:
-          SILKIT_SOURCE_URL: ${{ github.workspace }}/sil-kit
-          SILKIT_PKG_URL: ${{ github.workspace }}/sil-kit-pkg
           DEBFULLNAME: ${{ inputs.debian_fullname }}
           DEBEMAIL: ${{ inputs.debian_fullname }}
           CI_RUN: "1"
+          BUILD_SETTINGS: |
+              {
+
+                  "SilKitInfo": {
+                      "url": "sil-kit",
+                      "ref": "",
+                      "recursive": true,
+                      "is_local": true
+                  },
+                  "package_repo_path": "sil-kit-pkg",
+                  "version": {
+                      "major": 4,
+                      "minor": 0,
+                      "patch": 53,
+                      "suffix": "rc-1"
```

Comment:
do we have `rc-1` packages?

Reply:
We can if we want, at least for the .deb files. The version reported by silkit is still without the suffix. But I wanted something to distinguish non-release packages built for testing